voluseg/_steps/step0.py

```diff
@@ -144,48 +145,75 @@ def process_parameters(initial_parameters: dict) -> dict:
                 "the names of last directories in 'dir_input' must be sorted."
             )
 
+    remote = False
     volume_fullnames_input = []
     volume_names = []
-    for dir_input_h, dir_prefix_h in zip(input_dirs, prefix_dirs):
-        # get volume extension, volume names and number of segmentation timepoints
-        file_names = [i.split(".", 1) for i in os.listdir(dir_input_h) if "." in i]
-        file_exts, counts = np.unique(list(zip(*file_names))[1], return_counts=True)
-        ext = "." + file_exts[np.argmax(counts)]
-        volume_names_input_h = sorted([i for i, j in file_names if "." + j == ext])
-        volume_fullnames_input_h = [
-            os.path.join(dir_input_h, i) for i in volume_names_input_h
-        ]
+    if (".nwb" in dir_input) or ("https://" in dir_input):
+        if "https://" in dir_input:
+            remote = True
+            aux_list = [dir_input]
+        else:
+            aux_list = dir_input.split(":")
```

Comment:
In principle, we could use a similar approach for other 4D datasets (e.g. zarr).